t/discovery/nacos.t

```diff
@@ -32,15 +32,26 @@ deployment:
     config_provider: yaml
 discovery:
   nacos:
-      host:
-        - "http://127.0.0.1:8858"
-      prefix: "/nacos/v1/"
-      fetch_interval: 1
-      weight: 1
-      timeout:
-        connect: 2000
-        send: 2000
-        read: 5000
+    host:
+      - "http://127.0.0.1:8858"
+    prefix: "/nacos/v1/"
+    fetch_interval: 1
+    weight: 1
+    timeout:
+      connect: 2000
+      send: 2000
+      read: 5000
+    hosts:
+      - name: nacos3
+        host:
+          - "http://127.0.0.1:8868"
+        prefix: "/nacos/v1/"
+        fetch_interval: 1
+        weight: 1
+        timeout:
+          connect: 2000
+          send: 2000
+          read: 5000
```

Comment:
I don't think a mixed mode of single registry and multiple registries should exist, and the user should explicitly choose one or the other. This will also simplify our testing.

Reply:
do you propose removing `host` and introduce `hosts` as it's replacement? It would be a breaking change.

Reply:
@shreemaan-abhishek I am proposing: anyOf host or hosts, and prohibits the mixing of the two modes, so the following configuration is acceptable.
```
discovery:
  nacos:
    host: xxx
    prefix: xxx
```
Or,
```
discovery:
  nacos:
    - id: default
      host: xxx
      prefix: xxx
    - id: backup
      host: xxx
      prefix: xxx
```

Reply:
Yes this is much better.

Reply:
Yes this is much better. cc: @ShenFeng312